README.md

```diff
@@ -19,6 +19,42 @@
 
 ## Usage
 
+### Clojure CLI Tool
+
+Starting from Clojure CLI v1.11.1.1139,
+[tool](https://clojure.org/reference/deps_and_cli#tool_install)
+installation is supported.
+
+```bash
+# Install tool (latest version)
+clojure -Ttools install-latest :lib com.github.scarletcomply/license-finder :as license-finder
+
+# Install tool (certain version)
+clojure -Ttools com.github.scarletcomply/license-finder '{:git/sha "..."}' :as license-finder
```

Comment:
Once brach is merged to master, may be worth adding sha. Alternatively, this can be done after the release version is known with something like `{:git/tag "v0.3.0"}`